tests/trustyai/conftest.py

```diff
@@ -0,0 +1,111 @@
+from typing import Generator, Any
+
+import pytest
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.namespace import Namespace
+from ocp_resources.pod import Pod
+from ocp_resources.resource import get_client
+from ocp_resources.secret import Secret
+from ocp_resources.service import Service
+
+from tests.trustyai.utils.minio import MinioPod, MinioService, MinioSecret
```

Comment:
please add to this pr (else the code is broken)

Reply:
I removed these imports, since I am not using those classes anymore